test/JDBC/expected/constraint_table_usage-vu-prepare.out

```diff
@@ -0,0 +1,23 @@
+use master;
+go
+
+create database constraint_table_usage_vu_prepare_db;
```

Comment:
we do not need to create a separate database

Reply:
Resolved in latest update.